src/sentry/static/sentry/app/views/performance/realUserMonitoring/constants.tsx

```diff
@@ -0,0 +1,44 @@
+import {t} from 'app/locale';
+
+import {Vital, WebVital} from './types';
+
+export const NUM_BUCKETS = 50;
+
+export const PERCENTILE = 0.75;
+
+export const WEB_VITAL_DETAILS: Record<WebVital, Vital> = {
+  [WebVital.FP]: {
+    slug: 'fp',
+    name: t('First Paint'),
+    description: t(
+      'First paint refers to the point at which the first pixel renders on a screen after a user navigates to a web page.'
```

Comment:
@Zylphrex I added the description for first paint (FP) here.

Reply:
The language here feels like it deviates from the other three descriptions.

Reply:
I agree. But it's an accurate description.